sources/core/Hashes.h

```diff
@@ -57,4 +58,9 @@ inline unsigned int Hash(qint64 value)
     return static_cast<quint32>(value) + static_cast<quint32>(value >> 32);
 }
 
+inline unsigned int Hash(const std::bitset<4>& value)
```

Comment:
Again, I think it will be better to move this function to `SaveableOperators.h` file because the `Hashes.h` file is more about super-basic values, and `std::bitset` is more specific one.